gradle/balNativeLibProject.gradle

```diff
@@ -1,36 +1,4 @@
-apply from: "$rootDir/gradle/javaProject.gradle"
-
-configurations {
-    baloCreat
-    baloImplementation {
-        transitive false
-    }
-}
-
-dependencies {
-    baloCreat project(':lib-creator')
-    annotationProcessor project(':ballerina-lang')
-}
-
-compileJava {
-    def generatedSources = "$buildDir/generated-from-annot"
-    def generatedOutputDir = file("$generatedSources")
-    def pkgName = project.name.replace("ballerina", "").replaceAll("-","")
-    options.compilerArgs += ["-AnativeEntityProviderClass=StandardNativeElementProvider",
-                             "-AnativeEntityProviderPackage=org.ballerinalang.${pkgName}.generated.providers",
-                             "-s", "$generatedSources"]
-
-    doFirst {
-        generatedOutputDir.exists() || generatedOutputDir.mkdirs()
-    }
-}
-
-
-task createBalHome(type: org.gradle.api.tasks.Copy) {
-    from configurations.baloImplementation
-    into "$buildDir/lib"
-
-}
+apply from: "$rootDir/gradle/testableBaloProject.gradle"
```

Comment:
shall we rename this to 'baloProject'?

Reply:
Shall I rename to `javaProjectWithExtBalo.gradle` since this will not be used for balo projects, but only for java projects that need to include balos from other modules?